capstone/src/main/webapp/authenticate.js

```diff
@@ -59,11 +51,12 @@ function initClient() {
         discoveryDocs: DISCOVERY_DOCS,
         scope: SCOPES
     }).then(function () {
+        // Set ID_TOKEN and dispatch event
+        ID_TOKEN = gapi.auth2.getAuthInstance().currentUser.get().getAuthResponse().id_token;
```

Comment:
I thinking you can do: `window.ID_TOKEN` to make it clear that this is a global variable. In JS, any variable declared in the global context is automatically added to window.

(same for uses of ID_TOKEN below.

Reply:
Got it.